### ISSUE TITLE
TA Checkin #1

### ISSUE THREAD

**Issue by StudentJamesChen** [StudentJamesChen/ladybug#11]
1. How long are we meeting? Where is the meeting at?
This meeting is about 20 minutes long and held in the Senior Design lab (HEC-102). Be on time. The professors have talked about this extensively. You know what parking is like, you know you should be accounting for it. Also this is just a courtesy to other students as there may be back-to-back meetings, so we need to stick to the time allotted as closely as possible. Be sure to go in the lab and look for the TA you're meeting with.

2. Does the whole group have to be present?
YES. Unless you have otherwise received permission, every group member should be present in the lab for your meeting.

3. What are you looking for?

4. When are we meeting? Wednesday next week (9/25)

Understanding of the project (purpose, requirements/goals) and the initial plan for the project (initial idea for implementation and resources needed, particularly ones that need to be set up for you, e.g. elevated access to particular workstations, access to Github/Design Document of previous team(s), etc.)
Team distribution of labor (selected project manager, roles assigned, team weekly meetings scheduled)
Communication with the sponsors (project expectations of sponsor, regular sponsor meetings scheduled)
Progress on setting up your Product/Sprint Backlog (have your Jira board up for us to look at)
TL;DR: Do you understand what your project is? Do you understand the expectations for your project from your sponsor? Have you set up structure within your team and with your sponsor? Do you have everything you need to get started? Have you gotten started?

All in all, we just want to see that you’re making foundational progress towards your project goals. We want you to succeed! We are here to help you succeed. If you have any questions or worries, please bring them up to us! We can’t guarantee you an answer on the spot but we can definitely help point you in the right direction. See you at your TA Check-In!
